python/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "valkey-glide"
-version = "255.255.255"
-edition = "2021"
+version = "0.1.0"
```

Comment:
Doesn't this affect release artifacts?

Reply:
No, that's a dummy version of the native lib. Not related to the version of package (client).
I aligned this with other clients.